doc/ErlangFormatterComparison.md

```diff
@@ -33,12 +33,13 @@ One of the biggest lacking features with current Erlang formatters in handling o
 
 `erlfmt` forked the erlang parser to make sure that it can handle macros and can handle all of the following macros and format them.
 
-```erlang formatted macros
+```erlang
 -module(macros).
 
 -include_lib("stdlib/include/assert.hrl").
 
 -define(MACRO(), object).
+
 `argument`(?MACRO()) -> ok.
```

Comment:
Not sure how backticks got here
```suggestion
argument(?MACRO()) -> ok.
```

Reply:
Somehow I thought that was part of, oops.
I thought it was strange and tested it will erlfmt and it still worked :D